src/store/ChartStore.js

```diff
@@ -274,6 +283,53 @@ class ChartStore {
         this.feed.onComparisonDataUpdate(this.updateComparisons);
     }
 
+
+    /**
+     * Get tradeTimes if not loaded yet
+     * OR update the active symbols by comapring open time
+     */
+    @action.bound updateMarketClosedStatus(api) {
+        const todayUtc = (new Date()).getTime();
+        const timeToEpochGMT = (hour) => {
+            const dateStr = new Date().toISOString().substring(0, 11);
+            return new Date(`${dateStr}${hour}Z`).getTime();
+        };
+
+
+        if (this.tradeTimes.length && this.tradeTimesCapture.getDate() === (new Date()).getDate()) {
+            this.tradeTimes.forEach((market) => {
+                market.submarkets.forEach((submarket) => {
+                    submarket.symbols.forEach((symbol) => {
+                        const foundSymbol = this.activeSymbols.find(item => item.symbol === symbol.symbol);
+                        if (foundSymbol) {
+                            let isOpen = false;
+                            for (let i = 0; i <= symbol.times.open.length; i++) {
+                                if (
+                                    symbol.times.open[i] &&
+                            symbol.times.close[i] &&
```

Comment:
do you need to check this? You are already looping the contents of `open`, and `open.length` should be the same as `close.length` right?